README.md

```diff
@@ -193,3 +190,55 @@ r.Use(func (next http.Handler) http.Handler {
 	})
 })
 ```
+
+## Embedded Request Viewer Access Tokens
```

Comment:
@alexadrake I thought about the different use cases where you might want to get an access token (ie during a handler, or a method to render html output or as part of some other non handler scenario) and added a number of access patterns for it to the SDK, the Typescript SDK will also be able to support the below access patterns.

You can either get it from the controller via getting the SDK instance or using the singleton or developer created SDK instance.

As is this pattern doesn't work for the Java SDK but I was having a look and it wouldn't be too hard to refactor slightly to also support an SDK instance and provide similar access patterns. Though I think its worth a discussion on whether you think it would be worthwhile in the Java SDK